code/game/mecha/mech_fabricator.dm

```diff
@@ -219,7 +219,7 @@
   * Removes the overlay to show the fab working and sets idle power usage settings. Additionally resets the description and turns off queue processing.
   */
 /obj/machinery/mecha_part_fabricator/proc/on_finish_printing()
-	cut_overlay("fab-active")
+	cut_overlays("fab-active")

```

Comment:
`cut_overlays()` does not accept an iconstate as an argument. This isn't a good fix for whatever is going on. This is just passing in a string as the `priority` argument.